src/json/more_related_products0.json

```diff
@@ -0,0 +1,30 @@
+{
+  "items": [{
+    "id": 7,
+    "img": "/img/product1_640x426.jpg",
+    "name": "Apple",
```

Comment:
Can you give the items different names? e.g. Premium Apple

Reply:
done